### ISSUE TITLE
Identify Key Stakeholders

### ISSUE THREAD

**Issue by xKabbe** [xKabbe/star#4]
### Description

List and define the roles of all stakeholders involved in the project, including team members, advisors, and any external partners.

### Expected Actions

* [ ] Compile a list of stakeholders
* [ ] Document roles and responsibilities

### Definition Of Done

- [ ] Code implemented
- [ ] Tests implemented and passing
- [ ] Documentation implemented
- [ ] GitHub Actions are passing

**Comment by xKabbe** [comment]
This is a https://github.com/xKabbe/star/labels/duplicate of the issue:

* #6 

Therefore, this issue can be closed after #6 was implemented

**Comment by xKabbe** [comment]
Can be closed, since #6 was already implemented and we dont have any stakeholders for this project
